app/controllers/cases_controller.rb

```diff
@@ -196,10 +196,15 @@ def permitted_correspondence_types
     # @permitted_correspondence_types so that it's changed as an atomic
     # operation ... we don't want to possibly allow SAR or ICO types to be
     # permitted at any point.
-    types = current_user.managing_teams.first.correspondence_types.menu_visible.order(:name).to_a
-    types.delete(CorrespondenceType.sar) unless FeatureSet.sars.enabled?
-    types.delete(CorrespondenceType.ico) unless FeatureSet.ico.enabled?
-    types.delete(CorrespondenceType.offender_sar) unless FeatureSet.offender_sars.enabled?
-    @permitted_correspondence_types = types
+
+    if current_user.permitted_correspondence_types == [CorrespondenceType.offender_sar]
```

Comment:
Can we use the same logic for `can_add_case?` from Base.policy? below?
What if a Branston user can see other case types in the future?